qcodes/tests/dataset/test_measurement_context_manager.py

```diff
@@ -245,6 +245,24 @@ def test_log_includes_extra_info(caplog, meas_with_registered_param):
 
     assert "some extra info" in caplog.text
 
+def test_nested_measurement_throws_error(experiment, DAC, DMM):
```

Comment:
remove unused fixtures :)
```suggestion
def test_nested_measurement_throws_error(experiment):
```